tests/npm-tsoa-app/src/ccf.ts

```diff
@@ -0,0 +1,27 @@
+export interface Body<T> {
```

Comment:
Can we put this somewhere shared, rather than repeating it in each app?

Reply:
That's a tricky one. I thought about it, but I'd like to keep the test apps self-contained so that someone can just copy-paste them. I don't want to go down the path (yet) of creating a ccf npm package which could contain this stuff.

Reply:
Ok - the copy-paste argument wins me over for now.